pkg/types/types.go

```diff
@@ -32,6 +32,13 @@ var (
 		color.New(color.FgHiRed).SprintFunc(),
 		color.New(color.FgRed).SprintFunc(),
 	}
+	VulnerabilityTypes = []string{
+		"CVE",
+		"CWE",
+		"ELSA",
+		"OSVDB",
+		"USN",
+	}
```

Comment:
nit: this is just my preference.

```
type VulnerabilityType int

const (
	VulnerabilityTypeCVE VulnerabilityType = iota
	VulnerabilityTypeCWE
	VulnerabilityTypeELSA
	VulnerabilityTypeOSVDB
)

func (v VulnerabilityType) String() string{
	VulnerabilityTypes[v]
}
```

or

```
type VulnerabilityType int

const (
	VulnerabilityTypeCVE   VulnerabilityType = "CVE"
	VulnerabilityTypeCWE   VulnerabilityType = "CWE"
	VulnerabilityTypeELSA  VulnerabilityType = "ELSA"
	VulnerabilityTypeOSVDB VulnerabilityType = "OSVDB"
)

```

like the code of Severity.
https://github.com/aquasecurity/trivy-db/blob/c277fb3847d809715db8d75c29c246e0cb0bc1d3/pkg/types/types.go#L10-L18